code/src/components/Message.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { useEffect, useState } from 'react';
+import './Message.css';
```

Comment:
Let's remember about one empty line between semantically different type of imports